src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/alerts/AlertsSynchronisationService.kt

```diff
@@ -235,12 +235,48 @@ class AlertsSynchronisationService(
     val movedToNomsNumber = prisonerMergeEvent.additionalInformation.movedToNomsNumber
     val movedFromNomsNumber = prisonerMergeEvent.additionalInformation.movedFromNomsNumber
 
-    val alerts = nomisApiService.getAlertsToResynchronise(movedFromNomsNumber) ?: PrisonerAlertsResponse(emptyList())
+    synchronisePrisonerBookingMovedForPrisoner(
+      MoveBookingForPrisoner(
+        bookingId = bookingId,
+        offenderNo = movedFromNomsNumber,
+        whichPrisoner = WhichMoveBookingPrisoner.FROM,
+      ),
+    )
+
+    queueService.sendMessage(
+      messageType = SynchronisationMessageType.RESYNCHRONISE_MOVE_BOOKING_TARGET.name,
+      synchronisationType = SynchronisationType.ALERTS,
+      message = MoveBookingForPrisoner(
+        bookingId = bookingId,
+        offenderNo = movedToNomsNumber,
+        whichPrisoner = WhichMoveBookingPrisoner.FROM,
+      ),
+    )
```

Comment:
eventually when we receive this message will process the other prisoner record if needed, e.g. they are INACTIVE